lib/tutorials/en_US/getting-started.md

```diff
@@ -83,51 +79,48 @@ The `method` parameter can be any valid HTTP method, array of HTTP methods, or a
 
 We've proven that we can start a simple Hapi app with our Hello World application. Next, we'll use a plugin called **inert** to serve a static page. Before you begin, stop the server with **CTRL + C**.
 
-To install inert run this command at the command line: `npm install --save inert` This will download inert and add it to your `package.json`, which documents which packages are installed.
+To install [inert](https://github.com/hapijs/inert) run this command at the command line: `npm install --save inert` This will download [inert](https://github.com/hapijs/inert) and add it to your `package.json`, which documents which packages are installed.
 
-Add the following to your `server.js` file:
+Update the `init` function in your `server.js` file:
 
 ``` javascript
-server.register(require('inert'), (err) => {
-
-    if (err) {
-        throw err;
-    }
+const init = async () => {
+    await server.register(require('inert'));
 
     server.route({
         method: 'GET',
         path: '/hello',
-        handler: function (request, reply) {
-            reply.file('./public/hello.html');
+        handler: (request, h) => {
+            return h.file('./public/hello.html');
         }
     });
-});
 
+    await server.start();
+    console.log(`Server running at: ${server.info.uri}`);
+};
 
 ```
 
-The `server.register()` command above adds the inert plugin to your Hapi application. If something goes wrong, we want to know, so we've passed in an anonymous function which if invoked will receive `err` and throw that error. This callback function is required when registering plugins.
+The `server.register()` command above adds the [inert](https://github.com/hapijs/inert) plugin to your Hapi application. If something goes wrong, we want to know, so we've added a `try catch` that will receive any errors. It is recommended to always add a catch when working with promises to prevent errors from being swallowed.
 
-The `server.route`() command registers the `/hello` route, which tells your server to accept GET requests to `/hello` and reply with the contents of the `hello.html` file. We've put the routing callback function inside of registering inert because we need to insure that inert is registered _before_ we use it to render the static page. It is generally wise to run code that depends on a plugin within the callback that registers that plugin so that you can be absolutely sure that plugin exists when your code runs.
+The `server.route`() command registers the `/hello` route, which tells your server to accept GET requests to `/hello` and reply with the contents of the `hello.html` file.
```

Comment:
IMO the deleted sentence is quite useful to know:

> We've put the routing callback function inside of registering inert because we need to insure that inert is registered _before_ we use it to render the static page. It is generally wise to run code that depends on a plugin within the callback that registers that plugin so that you can be absolutely sure that plugin exists when your code runs.

There's no callback anymore but the ordering can still matter, for instance if you used the `{ file: { path: '...' } }` handler rather than `h.file()` you'd still have to ensure inert was registered first. 

Did you think this was too much information?